### ISSUE TITLE
Some checkpoint output images lose color and contrast.

### ISSUE THREAD

**Issue by chengkeng** [rosinality/stylegan2-pytorch#189]
Hi！
Mostly happen at checkpoints converted from stylegan-ada or official:
When I converted MetFaces_ADA.pkl to MetFaces_ADA.pt, the output graph in apply_factor.py is like this:
![2021-02-05 220612](https://user-images.githubusercontent.com/33344635/107043779-6c399800-67fe-11eb-9879-512e39a8a26d.png)
But when I use generate.py, the MetFaces_ADA.pt output image is normal:
![2021-02-05 220102](https://user-images.githubusercontent.com/33344635/107043883-8b382a00-67fe-11eb-9ee5-3effffca68ec.png)
File 550000.pt has similar situations, 
I tried to convert FFHQ.pkl and it was like this.
How can I fix them?
Thank you!

**Comment by mlyarthur** [comment]
Have you solved this problem? I am facing similar confusion that my results trained with this stylegan  is strange.
![100800](https://user-images.githubusercontent.com/29242717/121291956-b43b0980-c91b-11eb-833b-1e93de499af5.png)

**Comment by anuja13** [comment]
I have the same issue, @chengkeng @mlyarthur  could you solve it?

**Comment by rosinality** [comment]
@anuja13 Please check normalization before saving the image. Range should be [-1, 1].

**Comment by anuja13** [comment]
@rosinality Thank you for quick reply, I am saving in the range of [-1, 1], using the following code, but have blurred samples still, like the ones shown above.

grid = utils.save_image(
torch.cat([img1, img, img2], 0),
 f"{args.out_prefix}_index-{args.index}_degree-{args.degree}.png",
normalize=True,
range=(-1, 1),
nrow=args.n_sample )

**Comment by anuja13** [comment]
I figured the problem, for those facing this issue, set flag input_is_latent = False, while generating samples.

**Comment by dinarkino** [comment]
The fix is pretty simple. There is a misspelling in the code. You should use `value_range` instead of `range` in the `utils.save_image`
